src/screens/CreateCommunity/index.js

```diff
@@ -82,6 +83,14 @@ const CreateCommunity = () => {
     setName(value);
   };
 
+  React.useEffect(() => {
+    if (inputRef?.current) {
+      setTimeout(() => {
+        inputRef.current.focus();
+      }, 500);
+    }
+  }, []);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This `useEffect` hook is used to set focus on the input field after a delay of 500ms. It's a good way to ensure that the input field gets focused once the component has been mounted and rendered properly. However, it would be better to clear the timeout when the component unmounts to prevent potential memory leaks.
```diff
React.useEffect(() => {
    if (inputRef?.current) {
+       const timer = setTimeout(() => {
            inputRef.current.focus();
-       }, 500);
+       }, 500);
+       return () => clearTimeout(timer);
    }
}, []);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->